addons/mail/static/tests/qunit_suite_tests/components/message_tests.js

```diff
@@ -835,6 +835,46 @@ QUnit.test('chat with author should be opened after clicking on their avatar', a
     );
 });
 
+QUnit.test('chat with author should be opened after clicking on their name', async function (assert) {
+    assert.expect(4);
+
+    const pyEnv = await startServer();
+    const [threadId, resPartnerId] = pyEnv['res.partner'].create([{}, {}]);
+    pyEnv['res.users'].create({ partner_id: resPartnerId });
+    pyEnv['mail.message'].create({
+        author_id: resPartnerId,
+        body: 'not empty',
+        model: 'res.partner',
+        res_id: threadId,
+    });
+    const { click, createChatterContainerComponent } = await start();
+    await createChatterContainerComponent({
```

Comment:
use `openView` instead, this method is deprecated
see https://github.com/odoo/odoo/pull/94175